lib/utils/docker.coffee

```diff
@@ -244,3 +290,72 @@ exports.getDocker = (options) ->
 exports.getImageSize = (docker, image) ->
 	docker.getImage(image).inspectAsync()
 	.get('Size')
+
+hasQemu = ->
+	fs = require('mz/fs')
+
+	getQemuPath()
+	.then (qemu) ->
+		fs.stat(qemu)
+	.then ->
+		return true
+	.catch ->
+		return false
+
+getQemuPath = ->
+	resin = require('resin-sdk-preconfigured')
+	path = require('path')
+	fs = require('mz/fs')
+
+	resin.settings.get('binDirectory')
+	.then (binDir) ->
+		# The directory might not be created already,
+		# if not, create it
+		fs.access(binDir)
+		.catch (e) ->
+			if e.code == 'ENOENT'
+				fs.mkdir(binDir)
+			else
+				throw e
+		.then ->
+			path.join(binDir, 'qemu-execve')
+
+platformNeedsQemu = ->
```

Comment:
Oh, ok, I didn't realise OSX doesn't need it at all. In that case this makes totally sense :smile:.